core/vm/operations_verkle.go

```diff
@@ -1,160 +0,0 @@
-// Copyright 2024 The go-ethereum Authors
```

Comment:
Nothing about this file survived, which makes sense.
The `gas*(...)` style of apis can't work with what we need to do, so it makes sense that everything here went away (since was mostly moved to `op*(..)` above).